proxy/http2/Http2ConnectionState.cc

```diff
@@ -1160,16 +1170,8 @@ Http2ConnectionState::delete_stream(Http2Stream *stream)
 void
 Http2ConnectionState::release_stream(Http2Stream *stream)
 {
-  // Update stream counts
   if (stream) {
     --total_client_streams_count;
```

Comment:
Leave some comment why only `total_client_streams_count` need to be decremented here and others are not.